private-pools-example/README.md

```diff
@@ -0,0 +1,287 @@
+# Accessing private resources using Cloud Build Private Pools
+
+In this repository, we will demonstrate the below functionality of Cloud Build Private Pools:
```

Comment:
This sample demonstrates the following features of Cloud Build private pools:

Reply:
Updated.